mmedit/models/backbones/sr_backbones/ttsr_net.py

```diff
@@ -0,0 +1,71 @@
+import torch.nn as nn
+import torch.nn.functional as F
+
+from mmedit.models.common import ResidualBlockNoBN, make_layer
+
+
+def norm_conv_layer(in_channels, out_channels, stride=1):
```

Comment:
mmcv has both ConvModule and build_conv_layer, see examples in https://github.com/open-mmlab/mmclassification/blob/master/mmcls/models/backbones/resnet.py